docs/source/worker.rst

```diff
@@ -415,13 +415,20 @@ Dask workers are by default launched, monitored, and managed by a small Nanny
 process.
 
 .. autoclass:: distributed.nanny.Nanny
+   :members:
 
 
 API Documentation
 -----------------
 
-.. autoclass:: distributed.worker.TaskState
+.. autoclass:: distributed.worker_state_machine.TaskState
+   :members:
+
+.. autoclass:: distributed.worker_state_machine.UniqueTaskHeap
+   :members:
+
```

Comment:
I'm not sure if this should be publicly documented

Reply:
makes sense - removing it